lib/components/normal-components/Resonator.ts

```diff
@@ -0,0 +1,60 @@
+import { resonatorProps } from "@tscircuit/props"
+import { NormalComponent } from "../base-components/NormalComponent"
+import { Port } from "../primitive-components/Port"
+import { SourceSimpleResonator } from "circuit-json"
+import type { SchematicPortArrangement } from "@tscircuit/props"
+
+export class Resonator extends NormalComponent<typeof resonatorProps> {
+  get config() {
+    return {
```

Comment:
It is missing the default `schematicSymbolName` property.

Reply:
done.